--- a/player.go
+++ b/player.go
@@ -36,14 +36,13 @@ type Player struct {
 func newPlayer() *Player {
 	// Read initial volume from config
 	var initVol float64 = (viper.GetFloat64("volume") - 50.0) / 10.0
-	initVol := absVolume(viper.GetInt("volume"))
 
 	// making sure user does not give invalid volume
 	if initVol > 100 {
 		initVol = 100
 	} else if initVol < 1 {
 		initVol = 0
 	}
 
 	return &Player{volume: initVol}
 }
