--- a/playlist.go
+++ b/playlist.go
@@ -106,16 +106,17 @@ func newPlaylist() *Playlist {
 	playlist.SetInputCapture(func(e *tcell.EventKey) *tcell.EventKey {
 
 		currNode := playlist.GetCurrentNode()
 
 		audioFile := currNode.GetReference().(*AudioFile)
 
 		switch e.Rune() {
 
 		case ' ':
+			// Disable default key handler
 		case 'a':
 
 			name, _ := gomu.pages.GetFrontPage()
 
 			if name != "mkdir-popup" {
 				createPlaylistPopup()
 			}
