--- a/playlist.go
+++ b/playlist.go
@@ -530,24 +530,23 @@ func Ytdl(url string, selPlaylist *tview.TreeNode) {
 	go func() {
 
 		err := cmd.Run()
 		if err != nil {
 			timedPopup(" Error ", "Error running youtube-dl", getPopupTimeout(), 0, 0)
 			return
 		}
 
 		playlistPath := path.Join(expandTilde(dir), selPlaylistName)
 
-		downloadedAudioPath := downloadedFilePath(
 			stdout.Bytes(), playlistPath)
 
 		err = gomu.Playlist.AddSongToPlaylist(downloadedAudioPath, selPlaylist)
 
 		if err != nil {
 			log.Println(err)
 		}
 
 		downloadFinishedMessage := fmt.Sprintf("Finished downloading\n%s",
 			path.Base(downloadedAudioPath))
 
 		timedPopup(
 			" Ytdl ",
