--- a/playlist.go
+++ b/playlist.go
@@ -517,14 +517,13 @@ func (p *Playlist) FindAudioFile(audioName string) *AudioFile {
 }
 
 // download audio from youtube audio and adds the song to the selected playlist
-func Ytdl(url string, selPlaylist *tview.TreeNode) {
 
 	// lookup if youtube-dl exists
 	_, err := exec.LookPath("youtube-dl")
 
 	if err != nil {
 		timedPopup(" Error ", "youtube-dl is not in your $PATH", getPopupTimeout(), 0, 0)
 		return
 	}
 
 	dir := viper.GetString("music_dir")
