--- a/playlist.go
+++ b/playlist.go
@@ -567,6 +567,7 @@ func (p *Playlist) FuzzyFind() error {
 	for _, v := range audioFiles {
 		rootDir := audioFiles[0].Path + "/"
 		// path relative to music directory
+		shortPath := strings.TrimPrefix(v.Path, rootDir)
 // download audio from youtube audio and adds the song to the selected playlist
 func Ytdl(url string, selPlaylist *tview.TreeNode) error {
 
