--- a/playlist.go
+++ b/playlist.go
@@ -276,19 +276,20 @@ func populate(root *tview.TreeNode, rootPath string) {
 
 			audioLength, err := GetLength(path)
 
+			if err != nil {
 
 		audioFile := &AudioFile{
 			Name:        file.Name(),
 			Path:        path,
 			IsAudioFile: true,
 			Parent:      root,
 		}
 
 		child.SetReference(audioFile)
 
 		if file.IsDir() {
 			audioFile.IsAudioFile = false
 			populate(child, path)
 		}
 
 	}
